<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BabbleMainWindow</class>
 <widget class="QWidget" name="BabbleMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>说都不会话了</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTextEdit" name="plainText"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,2">
     <item>
      <widget class="QPushButton" name="encryptButton">
       <property name="text">
        <string>加密&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="decryptButton">
       <property name="text">
        <string>&lt;解密</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settingsButton">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="cipherText"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
